content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: 名字空间
+api_metadata:
+  - apiVersion: "v1"
+    kind: "Namespace"
```

Comment:
In the front-matter there seems to be some extra spaces compared to the English version of the file.

```suggestion
api_metadata:
- apiVersion: "v1"
  kind: "Namespace"
```

Reply:
Done, thanks